gifted_build/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "webpack -v",
     "build": "webpack --config webpack-prod.config.js && webpack --config webpack-server.config.js",
-    "start": "node src/server/dev-server.js",
+    "start": "node src/server/dev.js",
     "start:prod": "node dist/server.js"
```

Comment:
This should be now 
"start:prod": "node dist/prod.js"

with the new naming scheme